dali/operators/numba_function/numba_func.h

```diff
@@ -37,10 +37,19 @@ class NumbaFunc : public Operator<Backend> {
   void RunImpl(Workspace &ws) override;
 
  private:
-  uint64_t fn_ptr_;
+  uint64_t run_fn_;
```

Comment:
I'd add a note that in this operator we are keeping pointers as uint64_t to be able to pass them to the cfunc.
Perhaps add something like:
```
using NumbaPtr = uint64_t;
```
for better readability of the code.